parser/tidb/tidb_parser.go

```diff
@@ -0,0 +1,30 @@
+package tidb
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+
+	"github.com/pingcap/parser"
+	_ "github.com/pingcap/tidb/types/parser_driver"
```

Comment:
Do we need to import the driver?

Reply:
Followed https://github.com/pingcap/parser/blob/master/parser_example_test.go